afids_utils/tests/test_io.py

```diff
@@ -123,3 +267,56 @@ def test_write_fcsv(
                     str(afids_coords[idx][1]),
                     str(afids_coords[idx][2]),
                 )
+
+
+class TestAfidsToJson:
```

Comment:
It would be nice to (also) have a round-trip test where we use `afids_to_json` to write a JSON file then `get_afid` to check all the AFIDs.